### ISSUE TITLE
Script file concatenation for better code organization

### ISSUE THREAD

**Issue by danielgtaylor** [sintaxi/harp#127]
It would be nice to have a way to easily concatenate scripts so that you can split your js/coffee code into multiple files. How I would like to organize my code:

```
- main.js
| - main.coffee
| - users.coffee
| - utils.coffee
```

``` jade
script(src="/main.js")
```

What I currently have to do:

``` jade
script(src="/main.js/utils.js")
script(src="/main.js/users.js")
script(src="/main.js/main.js")
```

It looks like sprockets uses a special comment for this purpose:

``` coffeescript
#= include file.coffee
```

https://github.com/sstephenson/sprockets#the-include-directive

What are the chances of getting some kind of script preprocessing to allow this? It could potentially also be done with special folder names or JSON data files, so I'm not 100% set in the comment directive, it's just an idea. For example, any folder name that ends in `.js` could concatenate all files within it (but how do we manage order?).

**Comment by thesmart** [comment]
Maybe I'm a bit late to the party, but I started writing a partial solution to this issue.  One can specify a ```.man``` manifest file with the JavaScript and CoffeeScript files and folders they want included.  I hope this is useful to others:
https://github.com/sintaxi/terraform/compare/master...thesmart:master

I stopped development on Harp as a development static server and pivoted to using connect-assets as I think it is more suited to the requirements I have.
